### ISSUE TITLE
Safari 8.0.7 Bug: Opens link in both new tab AND current tab

### ISSUE THREAD

**Issue by ywecur** [cl3m/MiddleClick#18]

**Comment by monstrenyatko** [comment]
It might be because the application simulate the "middle click" on the 3-finger tap also.
I don't see the problem with iTerm but in Safari seems it works slightly differently and causes the issue.

**Comment by aucklander** [comment]
any solution on this, on chrome is does it too, opens in new tab and current tab. 
current solution for me to to enable 3 finger tap in options which will also trigger look up

with Bettertouchtool 3 finger click also starts up dictionary, so im trying to find a permanent option

**Comment by SilvM** [comment]
If you only tap with 3 fingers and not press, it works as it should

**Comment by Artanicus** [comment]
Thanks for the workaround @SilvM. The underlying issue still needs a fix though. I have tapping disabled so remembering to only tap for middle click is quite hard.

**Comment by cipri-tom** [comment]
Also on Firefox :( 

The above workaround works, but still not very convenient.

**Comment by WhyNotHugo** [comment]
Yup, the issue is that three-finger tap does a middle-click, but three-finger-press does a mouse1 click, so both events are triggered.
